spec/models/form_profile_spec.rb

```diff
@@ -368,6 +369,9 @@ def expect_prefilled(form_id)
       end
     end
 
+    context 'with a healthcare application form' do
+    end
```

Comment:
missing code or not needed?

Reply:
Ah, leftover from an earlier spec I wrote to test this. I'll take it out.